kibbeh/src/ui/Feed.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {
+  MouseEventHandler,
+  useState,
+  useEffect,
+  ReactNode,
+} from "react";
+import { RoomCardProps, RoomCard } from "./RoomCard";
+import { Button } from "./Button";
+import { isPast } from "date-fns";
+
+export interface FeedProps {
+  title: string;
+  actionTitle: string;
+  onActionClicked: MouseEventHandler<HTMLButtonElement>;
+  rooms: RoomCardProps[];
+  emptyPlaceholder: ReactNode;
+}
+
+export const Feed: React.FC<FeedProps> = ({
+  title,
+  actionTitle,
+  onActionClicked,
+  rooms,
+  emptyPlaceholder,
+}) => {
+  const [liveRooms, setLiveRooms] = useState<RoomCardProps[]>([]);
+  const [scheduledRooms, setScheduledRooms] = useState<RoomCardProps[]>([]);
+
+  useEffect(() => {
+    setLiveRooms(rooms.filter((room) => isPast(room.scheduledFor)));
+    setScheduledRooms(rooms.filter((room) => !isPast(room.scheduledFor)));
+  }, [rooms]);
```

Comment:
`Feed` don't use useEffect and store rooms in state. Take rooms as a prop. Also don't worry about scheduled rooms for now until the mockup is done for that